src/mono/msbuild/common/LibraryBuilder.targets

```diff
@@ -35,4 +41,35 @@
     </LibraryBuilderTask>
   </Target>
 
+  <Target Name="_GenerateBundle"
+          Condition="'$(BundlesAssemblies)' == 'true'">
+
+    <PropertyGroup>
+      <_AssembliesToBundleSourceFile>$(BundleDir)bundled_assemblies.c</_AssembliesToBundleSourceFile>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_AssembliesToBundle Remove="@(_AssembliesToBundle)" />
+      <_AssembliesToBundle Include="@(_AssembliesToBundleInternal)" />
+      <_AssembliesToBundle Include="$(_ParsedRuntimeConfigFilePath)" />
```

Comment:
This is not an assembly, so maybe the concept should be more generic?

Reply:
Modified to `_ResourceToBundle`